src/pages/components/toggletip/style.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Toggletip
+description:
+  Toggletips use the disclosure pattern to toggle the visibility of a popover.
+  They display additional, hidden information upon the click of a UI trigger
+  element and can contain interactive elements.
+tabs: ['Usage', 'Style']
+---
+
+## Color
+
+### Toggletip
+
+| Element        | Property         | Color token           |
+| -------------- | ---------------- | --------------------- |
+| Label          | text color       | `$text-secondary`     |
+| Trigger button | svg              | `$icon-secondary`     |
+| Container      | background-color | `$background-inverse` |
+| Text           | color            | `$text-inverse`       |
+
+**Interactive states**
+
+| State  | Element        | Property | Color token     |
+| ------ | -------------- | -------- | --------------- |
+| Hover  | trigger button | svg      | `$icon-primary` |
+| Active | trigger button | svg      | `$icon-primary` |
+| Focus  | trigger button | border   | `$focus`        |
+
+<Row>
+<Column colLg={8}>
+
+![Toggletip colors and interactive colors.](images/toggletip-style-1.png)
+
+</Column>
+</Row>
+
+## Typography
+
+| Element   | font-size (px-rem) | font-weight   | type token      |
+| --------- | ------------------ | ------------- | --------------- |
+| Body text | 14px / 0.875rem    | Regular / 400 | `$body-long-01` |
```

Comment:
```suggestion
| Body text | 14px / 0.875rem    | Regular / 400 | `$body-01` |
```

Using the new [v11 type styles](https://v11.carbondesignsystem.com/guidelines/typography/type-sets#body-styles).